README.md

```diff
@@ -1 +1,72 @@
-# azure-cosmosdb-mongodbshardedcluster
\ No newline at end of file
+# Install MongoDB Sharding Cluster
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fcjsingh8512%2Fazure-cosmosdb-mongodbshardedcluster%2Fusers%2Fchsi%2Fazuredeploy.json" target="_blank">
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.png"/>
```

Comment:
Did we fix the DeploytToAzure pipeline for this repo?

Reply:
Yes, You can deploy this template by clicking the DeployToAzure button